sys/abi/src/lib.rs

```diff
@@ -182,7 +182,7 @@ pub struct ULease {
     pub length: u32,
 }
 
-#[derive(Copy, Clone, Debug, FromBytes)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, FromBytes)]
```

Comment:
I think the one merge conflict on this branch is just that I also added these derives in #1661, but in a different order. Whoopsie.

Reply:
Yup! I've just removed that part of my change in favor of yours.